common/types/block.go

```diff
@@ -152,6 +154,21 @@ func (b *Block) Bytes() []byte {
 	return blkBytes
 }
 
+// Fields returns an array of LoggableFields for logging
+func (b *Block) Fields() []log.LoggableField {
+	return []log.LoggableField{
+		log.BlockID(b.ID().String()),
+		log.LayerID(uint64(b.LayerIndex)),
+		log.MinerID(b.MinerID().ShortString()),
```

Comment:
`b.MinerID()` returns `*signing.PublicKey`. If we make that type a loggable field it would simplify this line.

Another option, if you want to keep the `miner_id` label, is to add a `.Field(label string)` method to the type and then call it from here with the desired label.